src/business-logic/panel-integrations/skaarhoj-panel.ts

```diff
@@ -199,7 +185,7 @@ export class SkaarhojPanel implements Panel {
 
     // It's possible to get an id like "3.4", so the Math.floor is to turn that into "3" for now since we don't support multiple functions for a single button.
     const id: number = Math.floor(Number.parseFloat(match.groups.id!))
-    const inputConfiguration: InputConfiguration | undefined = this.panelLayoutConfiguration.inputConfigurations[id]
+    const inputConfiguration: InputConfiguration | undefined = this.getInputConfiguration(id + '')
```

Comment:
if you change line 187 to 
```typescript
const id: string = `${Math.floor(Number.parseFloat(match.groups.id!))}`
```

```suggestion
    const inputConfiguration: InputConfiguration | undefined = this.getInputConfiguration(id)
```

Or you could do this :D 

```suggestion
    const inputConfiguration: InputConfiguration | undefined = this.getInputConfiguration(`${id}`)
```

Reply:
Or if we never convert it to a number: https://github.com/tv2/sofie-tv2-server/pull/44#discussion_r1825476089

Reply:
It has been turned into a string in the comment that Anders linked.